### ISSUE TITLE
Is it possible to support module?

### ISSUE THREAD

**Issue by dcb9** [livebud/bud#207]
I'm going to launch a web application, which contains two modules, admin panel and user panel, but now I haven't seen any docs yet.

**Comment by matthewmueller** [comment]
Hey @dcb9, thanks for your interest in Bud! Do you mean two separate apps with different go modules?

**Comment by dcb9** [comment]
No, not that one, what I want is that two frontend modules in a single Bud project. The route of admin panel should be `/admin/users`, of user center should be `/` for index and `/home` for user home.
